ddl/db_integration_test.go

```diff
@@ -3060,11 +3045,14 @@ func TestAutoIncrementForceAutoIDCache(t *testing.T) {
 	// Cannot set next global ID to 0.
 	tk.MustExec("alter table t force auto_increment = 0;")
 	tk.MustExec("alter table t force auto_increment = 1;")
-	require.Equal(t, uint64(3), getNextGlobalID())
+	tk.MustQuery("show table t next_row_id").Check(testkit.Rows(
```

Comment:
when auto_id_cache=1, row id cache is still used, so it's not 3, instead, it's a bigger number